hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/sender/http/JettyMessageSenderTest.java

```diff
@@ -152,7 +163,14 @@ public void shouldSendRetryCounterInHeader() {
     @Test
     public void shouldSendAuthorizationHeaderIfAuthorizationProviderAttached() {
         // given
-        JettyMessageSender messageSender = new JettyMessageSender(client, address, Optional.of(m -> "Basic Hello!"), 1000, new DefaultHttpMetadataAppender());
+        HashMap<String,String> authData = new HashMap<String,String>();
```

Comment:
i don't need to create subscription here -> why not stay with simple function that just returns fixed text? looks like you try to test some other stuff here